### PR TITLE
add pool to bgbouncer

### PR DESCRIPTION
**Pull request by ShinIce**
hey @kmoppel 👋🏻 

here is the PR for "pools" in pgbouncer, probably not perfect but working 😁  but two open points:

- I don't use go, so probably the part on the pgwatch2.go could be done better
- exporting the grafana dashboard breaks your template, so I think posting here the query would be "safer"
`select
  $__timeGroup(time, $__interval),
  c as "cl_active"
from (
  select 
    (data->>'cl_active')::int8 as c,
    time
  from pgbouncer_pools
  where dbname = '$dbname' and $__timeFilter(time)
            and data->>'user' = 'CHANGEME'
) x`
basically I have 3 queries up: cl_active, sv_active and sv_idle . placed right to the cybertec banner.
image

let me know what you think 😉 

cheers
Shin

### CODE REVIEW

**Comment by kmoppel** [comment]
Hey @ShinIce, finally got to the PR :) Thanks for the idea, I think I get what you're trying to achieve! And yes, best to handle it from the collector with some special behaviour similar to the current "pgbouncer_stats" metric.

I've tested your changes and it works indeed! My only naggle - database + user from ["show pools"](http://www.pgbouncer.org/usage.html#show-pools) is a "tag" type information that one would probably group by, and maybe search quickly for a specific value.

Would you be willing to alter the code a bit to just prepend the 2 fields with "tag_"? The framework will take care of the rest :)

**Comment by ShinIce** [comment]
hey @kmoppel 👋🏻 

yeah sure, how should I change it to fit in your idea? let me know all the changes and I'll test them out ;)

**Comment by kmoppel** [comment]
@ShinIce I think we can just add it to the FilterPgbouncerData function with some if-else. One needs to add a metric name input param though. Thanks!

**Comment by pashagolub** [comment]
Hey @ShinIce,

should we close or update this PR?

Thanks in advance!

**Comment by ShinIce** [comment]
Hi @pashagolub ,

sorry for the delay but no, will try to fix it this week or next week.

**Comment by pashagolub** [comment]
Sure, no problem. 

Let us know if you need any help with it.

**Comment by pashagolub** [comment]
Outdated. Thanks for the efforts @ShinIce! Feel free to reopen this PR in case of need! 🤝
